doc/src/vpicUnits.tex

```diff
@@ -37,41 +51,156 @@
 		Q &= l_{\mathrm{deck}}\left(\frac{c}{c_{\mathrm{deck}}}\right)^2\frac{\varepsilon_0}{\varepsilon_{0,\mathrm{deck}}}\frac{r_{\alpha,\mathrm{deck}}}{r_{\alpha}}.
 	\end{align}
 
-	\noindent The electric and magnetic field conversion factors $E$ and $B$ are therefore given by
+	\noindent The electric and magnetic field conversion factors $E$ and $B$ are then given by
 
 	\begin{align}
 		E &= \frac{ML}{QT^2} = \frac{1}{l_{\mathrm{deck}}}\frac{r_{\alpha,\mathrm{deck}}}{r_{\alpha}}\left(\frac{c}{c_{\mathrm{deck}}}\right)^2 \\
 		B &= E\frac{T}{L} = \frac{1}{l_{\mathrm{deck}}}\frac{r_{\alpha,\mathrm{deck}}}{r_{\alpha}}\frac{c}{c_{\mathrm{deck}}}
 	\end{align}
 
-	As an example of how this might work, one possible choice of units is $l_{\mathrm{deck}} = c/\omega_{\mathrm{pe}}$, $c_{\mathrm{deck}} = 1$, $\varepsilon_{0,\mathrm{deck}} = 1$ and $(q_e/m_e)_{\mathrm{deck}} = -1$. This gives us
+	\subsection{Particle Momenta}
 
-	\begin{align}
-		L &= \frac{c}{\omega_{\mathrm{pe}}}, \\
+	Particle momenta are the only user-facing exception to the unit system defined above. The momenta are normalised to produce a dimensionless momentum
+
+	\begin{equation}
+		u \equiv \frac{p}{mc}.
+	\end{equation}
+
+	This dimensionless momentum is used for example when loading particles, however the momentum density given in the hydro dumps is in the unit system defined above.
+
+	\subsection{Common Unit Systems}
+
+	Various common choices are made for the unit system, and we discuss a few of them here. In general, aside from ensuring that values do not exceed the limits for single-precision floating point numbers, it is often desireable to choose the unit system to ease interpretation of simulation results.
+
+	\subsubsection{Electrostatic Plasma Units}
+
+	In an electrostatic problem simulating an electron plasma the natural length and time-scales are the Debye length $\ldebye$ and inverse plasma frequency $\omegap^{-1}$. These are also the typical scales for the grid and time-step. Then choosing the electron charge/mass ratio so that $e/m_e$ becomes unity reduces the equation of motion of an electron to
+
+	\begin{equation*}
+		\ddot{\vec{r}} = \vec{E}.
```

Comment:
This is misleading.  VPIC will not change the math it does based on the unit system.  It needs to be clear that this is motivation for using electrostatic units in particular decks.

Reply:
I'm not sure I agree that this is a problem. I get what you're saying in that there won't necessarily be fewer multiplications going on in the code, but the differential equation correctly models what the outcome of the VPIC calculations will be (assuming the problem really is electrostatic, and non-relativistic, which I suppose I could clarify).